Update verify.mdx (#1014)
WalletConnect/walletconnect-docs: docs/web3wallet/verify.mdx

import PlatformTabs from '../components/PlatformTabs'
import PlatformTabItem from '../components/PlatformTabItem'

# Verify API

Verify API is a security-focused feature that allows wallets to notify end-users when they may be connecting to a suspicious or malicious domain, helping to prevent phishing attacks across the industry.

> Once the wallet knows whether the end-user is on uniswap.com or eviluniswap.com, it can help its users detect potentially harmful connections through Verify's combined offering of WalletConnect’s domain registry and industry-leading security tools.

Author: ```suggestion
Once a wallet knows whether an end-user is on uniswap.com or eviluniswap.com, it can help them to detect potentially harmful connections through Verify's combined offering of WalletConnect’s domain registry and industry-leading security tools.
```

For those looking to enable Verify on the app side, check out our reference guide [here.](../cloud/verify)

When a user initiates a connection with an application, Verify API enables wallets to present their users with four key states that can help them determine whether the domain they’re about to connect to might be malicious. 

These are:

> ![Verify Banner](/img/verify-banner.png)

Author: This also needs to be changed to the one [here](url)
